packages/website/ts/components/documentation/docs_top_bar.tsx

```diff
@@ -0,0 +1,155 @@
+                iconUrl: '/images/developers/chat_icon.svg',
+                textStyle: { color: colors.lightLinkBlue, fontWeight: 'bold' },
+            },
+        ];
+        return (
+            <Container height={80}>
+                <Container className="flex items-center lg-pt3 md-pt3 sm-pt1 relative" width="100%">
+                    <Container className="col col-2 sm-hide xs-hide">
+                        <Link
+                            to={WebsitePaths.Home}
+                            style={{ color: colors.linkSectionGrey }}
+                            className="flex items-center text-decoration-none"
+                        >
+                            <i className="zmdi zmdi-chevron-left bold" style={{ fontSize: 16 }} />
+                            <Container paddingLeft="8px">
+                                <Text fontSize="16px" fontColor={colors.linkSectionGrey}>
+                                    0xproject.com
+                                </Text>
+                            </Container>
+                        </Link>
+                    </Container>
+                    <Container className="col col-4 md-hide sm-hide xs-hide" />
+                    <Container className="col col-6 md-pl4 md-ml4 sm-hide xs-hide">
+                        <Container className="flex items-center justify-between right" width="300px">
+                            {this._renderMenuItems(menuItemInfos)}
+                        </Container>
+                    </Container>
+                    <Container className="lg-hide md-hide">
+                        <DocsLogo height={30} containerStyle={{ paddingTop: 6, paddingLeft: 18 }} />
+                    </Container>
+                    <Container className="md-hide lg-hide absolute" right="18px" top="12px">
+                        <i
+                            className="zmdi zmdi-menu"
+                            style={{
+                                color: colors.grey700,
+                                fontSize: 30,
+                                cursor: 'pointer',
+                            }}
+                            onClick={this._onMenuButtonClick.bind(this)}
+                        />
+                    </Container>
+                </Container>
+                <Container width={'100%'} height={'1px'} backgroundColor={colors.grey300} marginTop={'11px'} />
+                {this._renderDrawer()}
+            </Container>
+        );
+    }
+    private _renderMenuItems(menuItemInfos: MenuItemInfo[]): React.ReactNode {
+        const menuItems = _.map(menuItemInfos, menuItemInfo => {
+            return (
+                <a
+                    key={`menu-item-${menuItemInfo.title}`}
+                    href={menuItemInfo.url}
+                    target="_blank"
+                    className="text-decoration-none"
+                    style={{
+                        fontSize: 16,
+                    }}
```

Comment:
nit: I would remove this inline style and just wrap `menuItemInfo.title` in a `Text`